.github/workflows/label-approved.yml

```diff
@@ -0,0 +1,18 @@
+name: Update Pull Request Approval Labels
+on:
+  pull_request_review:
+  pull_request:
+    types: [synchronize]
+
+permissions:
+  issues: write
```

Comment:
I'm surprised this is supposed to work with `issues` instead of `pull-requests`, but that may well be some scoping trick, where a PR is a special issue.

https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs#overview

> Work with pull requests. For example, pull-requests: write permits an action to add a label to a pull request. For more information, see "[Permissions required for GitHub Apps](https://docs.github.com/en/rest/overview/permissions-required-for-github-apps?apiVersion=2022-11-28#repository-permissions-for-pull-requests)."

Reply:
Thanks, I didn't look for the docs and I think I remembered from previous experiences with GH's API that the issues permission bit was enough here. Maybe actions are different?

The second link in there isn't really clear on whether both are needed or just one, but the `labels` workflow currently in nixpkgs seems to work fine with just `pull-requests: write`. Updating.